.eslintrc

```diff
@@ -288,6 +289,30 @@
       }
     ],
     "@typescript-eslint/no-for-in-array": "error",
+    "@typescript-eslint/consistent-indexed-object-style": "off",
+    "@typescript-eslint/no-floating-promises": "off",
+    "@typescript-eslint/require-await": "off",
+    "@typescript-eslint/no-unsafe-argument": "off",
+    "@typescript-eslint/no-unsafe-enum-comparison": "off",
+    "@typescript-eslint/prefer-nullish-coalescing": [
+      "off",
+      {
+        "ignoreConditionalTests": true,
+        "ignoreTernaryTests": false,
+        "ignoreMixedLogicalExpressions": true,
+        "ignorePrimitives": {
+          "boolean": true,
+          "number": false,
+          "string": true
+        }
+      }
+    ],
+    "@typescript-eslint/no-unsafe-assignment": "off",
+    "@typescript-eslint/no-unsafe-call": "off",
+    "@typescript-eslint/no-unsafe-member-access": "off",
+    "@typescript-eslint/no-unsafe-return": "off",
+    "@typescript-eslint/unbound-method": "off",
```

Comment:
If it weren't for it flagging lodash's stubs/noop, this one wouldn't be that bad. It does flag

https://github.com/DestinyItemManager/DIM/blob/5fdc84262ab820416136f7695a1b5cafd334009e/src/app/stream-deck/async-module.ts#L239

which in my limited understanding seems like it catches a bug here?

Reply:
Yeah I think I'll swap out noop for our own version with proper types.